### PR TITLE
Webbrowser fixes

### PR DESCRIPTION
**Pull request by ResearchDaniel**
- Windows: Prevent console windows from popping up
- Mac/linux: Removed libcef_lib since it seem to be windows only

### CODE REVIEW

File: modules/webbrowser/properties/propertycefsynchronizer.cpp [hunk 1]
```diff
@@ -33,6 +33,13 @@
 #include <modules/webbrowser/properties/ordinalpropertywidgetcef.h>
 #include <modules/webbrowser/properties/stringpropertywidgetcef.h>
 
+#include <modules/webbrowser/webbrowsermodule.h>
+
+#include <warn/push>
+#include <warn/ignore/all>
+#include "include/cef_parser.h"
```
Comment (petersteneteg) [inline modules/webbrowser/properties/propertycefsynchronizer.cpp:40]: generally "include" would not be in the header path. Do you have the correct include paths setup?

File: modules/webbrowser/webbrowsermodule.cpp [hunk 2]
```diff
@@ -162,4 +168,70 @@ WebBrowserModule::~WebBrowserModule() {
...
 }
+    
+std::string WebBrowserModule::getDataURI(const std::string& data, const std::string& mime_type) {
+    return "data:" + mime_type + ";base64," +
+    CefURIEncode(CefBase64Encode(data.data(), data.size()), false)
+    .ToString();
+}
+    
+std::string WebBrowserModule::getCefErrorString(cef_errorcode_t code) {
+#define CASE(code) \
```
Comment (petersteneteg) [inline modules/webbrowser/webbrowsermodule.cpp]: why use a define for this small snippet?

File: modules/webbrowser/webbrowsermodule.h [hunk 3]
```diff
@@ -34,6 +34,17 @@
 #include <inviwo/core/common/inviwomodule.h>
 #include <inviwo/core/util/timer.h>
 
+
+#include <warn/push>
+#include <warn/ignore/all>
+
+#include "include/internal/cef_types.h"
```
Comment (petersteneteg) [inline modules/webbrowser/webbrowsermodule.h:41]: same strange path...

**Review by petersteneteg — APPROVED** [review]
cant't really say I understand this, but it looks more or less reasonable

File: modules/webbrowser/properties/propertycefsynchronizer.cpp (hunk 1 above)
Comment (ResearchDaniel) [reply modules/webbrowser/properties/propertycefsynchronizer.cpp:40]: Use same as CEF examples, so yes :)

File: modules/webbrowser/webbrowsermodule.cpp (hunk 2 above)
Comment (ResearchDaniel) [reply modules/webbrowser/webbrowsermodule.cpp]: Better suggestion?

**Comment by ResearchDaniel** [comment]
@petersteneteg PR became a bit more involved since I wanted to make the new CEF version work on Mac as well. Have now verified that it also works on Windows

File: modules/webbrowser/webbrowsermodule.cpp (hunk 2 above)
Comment (petersteneteg) [reply modules/webbrowser/webbrowsermodule.cpp]: just write out the code is's only a few chars more...

**Pull request merged**